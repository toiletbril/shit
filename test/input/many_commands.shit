../shit -AE -c "echo 1" -c "echo 2"