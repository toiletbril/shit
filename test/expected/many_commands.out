[Sequence]
  [SequenceNode Simple]
    [Exec "../shit -AE -c echo 1 -c echo 2"]
[Sequence]
  [SequenceNode Simple]
    [Exec "echo 1"]
1
[Code 0]
[Sequence]
  [SequenceNode Simple]
    [Exec "echo 2"]
2
[Code 0]
[Code 0]
